projects/sightsea/sightsea.json

```diff
@@ -0,0 +1,45 @@
+{
+  "id": "sightsea",
+  "slug": "sightsea",
+  "name": "sightsea",
+  "category": "Social",
+  "coinGeckoId": "chainlink",
```

Comment:
I don't think CoinGeckid is correct, do you want to delete that?

Reply:
@themacexpert Thanks for pointing out. That was a mistake from copying a template, I'm so sorry about it.